Sources/BraintreeThreeDSecure/BTThreeDSecureClient.swift

```diff
@@ -350,10 +349,6 @@ import BraintreeCore
                 parameters: requestParameters
             ) { body, _, error in
                 if let error = error as NSError? {
-                    if error.code == BTCoreConstants.networkConnectionLostCode {
```

Comment:
Do we no longer want to send the `networkConnectionLost` event here?

Reply:
We have a ticket for [networkConnectionLost](https://paypal.atlassian.net/browse/DTBTSDK-2711). We realized that we were missing them so we added them to the Excel sheet listing all events. I can just add it for this module here.

Reply:
That makes sense. Yeah I think for the modules we have PRs for we can make sure they are added. Then in the followup worry about just the missing ones that we didn't catch the first time.